t/ui/02-list-group.t

```diff
@@ -153,5 +158,19 @@ subtest 'group_glob and not_group_glob' => sub {
     };
 };
 
+subtest 'filter for not_groupid' => sub {
+    $schema->resultset('Jobs')->create({
+            %$template,
+            id => 99955,
+            group_id => undef
+          });
```

Comment:
```suggestion
    $schema->resultset('Jobs')->create({%$template, id => 99955, group_id => undef});
```
(Unless tidy force you. You could also just modify `$template` which would probably be faster.)